src/mesido/esdl/profile_parser.py

```diff
@@ -125,6 +128,15 @@ def read_profiles(
                         values=values,
                         ensemble_member=ensemble_member,
                     )
+                    # Check if the installed heat demand capacity is sufficient
+                    if "heat_demand" in component_type:
```

Comment:
I prefer below, just in case we might result in adding an asset which has this string included in their name in the future: 
```suggestion
                    if "heat_demand" == component_type:
```
Ohh maybe also add "cold_demand", to be future proof

Reply:
Good spot thanks. I will update

Reply:
Resolved

Reply:
The code has been updated with cold_demand check